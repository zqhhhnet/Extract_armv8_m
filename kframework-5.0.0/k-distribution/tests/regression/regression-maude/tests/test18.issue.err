[Warning] Parser: Parsing ambiguity. Arbitrarily choosing the first.
1: B ::= "a" 
   a
2: A ::= "a" 
   a
	File: tests/regression/negative-test/programs/test.issue
	Location: (1,1,1,2)
	Compilation Phase: Ambiguity filter

