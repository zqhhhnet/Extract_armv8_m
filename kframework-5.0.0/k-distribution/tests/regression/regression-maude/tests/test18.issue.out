<T>
    <k>
        a
    </k>
    <j>
        .Bag
    </j>
    <one>
        .Set
    </one>
    <two>
        .Set
    </two>
</T>
